cell_test.go

```diff
@@ -0,0 +1,122 @@
+package tcell
```

Comment:
This file is useful for benchmarking and comparing, but I think we should ditch it from the final release.  I don't think we need to keep the history that it represents.

Reply:
Whatever works for you, I have no strong feelings here :)